js/validate.js

```diff
@@ -0,0 +1,31 @@
+const form = document.getElementById('my-form-button');
```

Comment:
first i noticed is i can submit your form without filling anything, lets fix that also the whole form validation should rely on js